.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx --no-install lint-staged
+npx lint-staged
```

Comment:
What's the difference between these two?

Reply:
Sorry bhaiiya I don't really know, followed the steps, I also thought about **npx --no-install lint-staged**, but thought might still add this using the command -> **npx husky add .husky/pre-commit "npx --no-install lint-staged"** as well, in order to follow the steps completely and make sure things worked this time.

Reply:
@chirayumit001 
Do one thing.  Try removing anyone and see what happens. If the result is same, revert it back. Do this for both. Once done, let me know. We can then go ahead with the merge.

Reply:
See [this](https://javascript.plainenglish.io/yes-its-npx-not-npm-the-difference-explained-58cbb202ec33) for the difference above.

Reply:
> @chirayumit001 Do one thing. Try removing anyone and see what happens. If the result is same, revert it back. Do this for both. Once done, let me know. We can then go ahead with the merge.

Done bhaiya, the result is same in both cases, it is working with either of them.

Reply:
@chirayumit001 
Just try one more thing. Try changing the command to `npm run lint-staged`. If it works, nice. If not, do the below.
Just commit the changes for --no-install one and remove the other.

Final change and then we can merge.

Reply:
@K-Kumar-01 
npm run command failed bhaiya, added --no--install command and removed other one as instructed